### PR TITLE
Issue#1990 Use Etc.getpwn.name instead of Etc.getlogin #1990

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1202238367235600) by [Unito](https://www.unito.io)

### CODE REVIEW

File: lib/tasks/rake_helper.rb [hunk 1]
```diff
@@ -188,7 +188,7 @@ def image_name
   end
 
   def user
-    @user ||= Etc.getpwnam(Etc.getlogin)
+    @user ||= Etc.getpwnam(Etc.getpwuid.name)
```
Comment (treydock) [inline lib/tasks/rake_helper.rb:191]: Both `getpwnam` and `getpwuid` return a `Passwd` object I believe so you might be able to simplify with this:
```suggestion
    @user ||= Etc.getpwuid
```

**Review by johrstrom — CHANGES_REQUESTED** [review]
@SaumyaBhushan Trey left some suggestions which I think are good. Can you make the changes?

**Comment by johrstrom** [comment]
Also @SaumyaBhushan thanks for opening the pull request to begin with! Happy to help with this and any other you may open or want to.

**Comment by SaumyaBhushan** [comment]
Changes have been committed. Please review the changes.

**Review by johrstrom — APPROVED** [review]
LGTM. Thanks again for the pull request. Just comment on any other ticket you'd like or are interested in completing.

**Pull request merged**